github/github.md

```diff
@@ -54,12 +54,36 @@ Instructions for Mac OSX
     - **DO NOT NAVIGATE AWAY FROM PAGE UNTIL YOU HAVE COPIED THE TOKEN AND PASTED IT SOMEWHERE IN A SAFE PLACE**
     - Paste the token in a note file for temporary storage
 - **Step 7**: Prompt GitHub authentication in the terminal
-  - Clone an existing repo using HTTPS
+
+  - Make a new repo in GitHub
```

Comment:
Maybe denote that it is the student's personal Github?

Reply:
Good note for clarity